src/auro-dropdown.js

```diff
@@ -219,13 +256,19 @@ class AuroDropdown extends LitElement {
       <div id="popover" class="popover" aria-live="polite" style=${`min-width: ${this.dropdownWidth}px;`}>
         <slot role="tooltip"></slot>
       </div>
-      <div id="trigger" class="trigger" data-trigger-placement="${this.placement}">
+      <div
+        id="trigger"
+        class="trigger"
+        role="button"
+        aria-label="${ifDefined(this.ariaLabel ? this.ariaLabel : undefined)}"
+        aria-labelledby="${ifDefined(this.ariaLabelledby ? this.ariaLabelledby : undefined)}"
```

Comment:
```suggestion
        aria-label="${ifDefined(this.ariaLabel)}"
        aria-labelledby="${ifDefined(this.ariaLabelledby)}"
```

I _think_ this should work instead - asumming ariaLabel and ariaLabelledby defaults to undefined

Reply:
You were right, the shorthand version does work. Thanks, @geoffrich.